### PR TITLE
Add formatting changes from black formatter

### PR DESCRIPTION
**Pull request by thefirstofthe300**
Adds formatting changes on top of https://github.com/taosdevops/harvest-reporting/pull/34

### CODE REVIEW

File: .github/workflows/python-test.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ jobs:
   test:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```
Comment (rmeyer-taos) [inline .github/workflows/python-test.yml:9]: Just wow.

File: hreporting/client.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        self.hooks = hooks
+        self.hours_left = hoursLeft
+        self.hours_total = hoursTotal
+        self.hours_used = hoursUsed
+        self.name = name
+        self.percent = percent
+        self.template_id = templateId
+
+
+class SendGridClient(HarvestClient):
```
Comment (rmeyer-taos) [inline hreporting/client.py:24]: Should verify and kill this thing.

Comment (thefirstofthe300) [reply hreporting/client.py:24]:
```/workspaces/harvest-reporting on ds/fix-email-hooks !1                                                                                                                  at 06:33:30 PM
> grep -r SendGridClient .    
/workspaces/harvest-reporting on ds/fix-email-hooks !1 ```

It's dead.

File: .github/workflows/python-test.yml (hunk 1 above)
Comment (thefirstofthe300) [reply .github/workflows/python-test.yml:9]: Right?

**Pull request merged**